core/new-gui/src/app/workspace/component/left-panel/versions-frame/versions-frame.component.ts

```diff
@@ -8,8 +8,8 @@ import { WorkflowVersionCollapsableEntry } from "../../../../dashboard/user/type
 @UntilDestroy()
 @Component({
   selector: "texera-version-frame",
-  templateUrl: "./versions-frame.component.html",
-  styleUrls: ["./versions-frame.component.scss"],
+  templateUrl: "versions-frame.component.html",
+  styleUrls: ["versions-frame.component.scss"],
 })
 export class VersionsFrameComponent implements OnInit {
```

Comment:
if we remove the `Frame` from operator menu component, do you want to consider rename this as well? For example, it could be `VersionsListComponent`. Just want to align your changes.

Reply:
Renamed to VersionsListComponent